scipy/stats/tests/test_stats.py

```diff
@@ -3110,6 +3108,36 @@ def test_ttest_ind_with_uneq_var():
                      ([0, np.nan], [1, np.nan]))
 
 
+def test_ttest_trimmed1():
```

Comment:
where do the test values come from?

also include corner cases: trim_perc = 0, 100

Reply:
The values comes from R. In R the test is called yuen T test (it is why I was insisting with the naming since in science it is known with that name). You can find online yuen t test calculators that use R and jupyter notebooks: https://rdrr.io/cran/PairedData/man/yuen.t.test.html